### PR TITLE
Update README.md version compatibility

### PR DESCRIPTION
**Pull request by mrfrenzy**
Adjust compatible versions of PHP and Mediawiki

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,11 +8,11 @@ MediaWiki Page: https://www.mediawiki.org/wiki/Extension:PHPBB_Auth
 REQUIREMENTS
 =================
 
-* PHP 7.3 or later
+* PHP 7.3 (not 7.4)
```
Comment (cpeel) [inline README.md]: This is incorrect. We are using PHP 7.4.3 at [Distributed Proofreaders](https://www.pgdp.net/c) and the plugin works fine there. I will look into the error you are seeing in https://github.com/Digitalroot-Technologies/MediaWiki_PHPBB_Auth/issues/58.

Comment (mrfrenzy) [reply README.md]: You are correct, the 7.4 related errors do not show themselves on Mediawiki 1.37 and older.

**Review by mrfrenzy — COMMENTED** [review]
Adjusted to remove comment about php 7.4
Also added phpBB 3.3.7 tested OK

**Comment by cpeel** [comment]
Thank you for these updates. I'll look into getting the plugin working with the new PluggableAuth 6.x -- will probably be a few weeks though.

**Pull request merged**